pkg/utility/utility.go

```diff
@@ -0,0 +1,65 @@
+package utility
+
+import (
+	"os"
+	"regexp"
+	"strings"
+
+	"github.com/tjarratt/babble"
+)
+
+const (
+	MaxUint = ^uint(0)
+	MaxInt  = int(MaxUint >> 1)
```

Comment:
This looks like it's used for the default value of the length flag, can we just define a defaultLength variable and use that instead for the default value?